vue-components/src/styles/mixins/Label.scss

```diff
@@ -8,6 +8,10 @@
   overflow-wrap: break-word;
   hyphens: auto;
 
+  &--disabled {
```

Comment:
mixins can accept parameters, as seen in line 1, so IMO it would be easier to add the logic regarding the label color into the mixin itself, rather than in each and every component.

Reply:
It does accept parameter but not dynamically so it can't change it on the fly (in case a label gets disabled by a user interaction). Right?

Reply:
hmm, that is very correct, I didn't think of that use case for some reason :sweat_smile: